BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -323,18 +312,10 @@ - (void)executeOperation
 
         if (self.error == nil && ![self isCancelled])
         {
-            //FIXME: remove shouldUseSessionTask and this condition after switching
-            //completely from NSURLConnection to NSURLSessionTask
-            if (self.shouldUseSessionTask == YES) {
-                if (self.sessionTask == nil) {
-                    self.sessionTask = [self createSessionTask];
-                }
-                [self.sessionTask resume];
-            } else {
-                self.connection = [[NSURLConnection alloc] initWithRequest:self.APIRequest delegate:self];
-                BOXLog(@"Starting %@", self);
-                [self startURLConnection];
+            if (self.sessionTask == nil) {
```

Comment:
I disagree. As long as we don't have a lazy instantiation accessor there is no need to use the ivar directly and in fact that could have undesirable side-effects. If that changes we'd have to review usage of the accessor, but until then using the property is the correct approach.